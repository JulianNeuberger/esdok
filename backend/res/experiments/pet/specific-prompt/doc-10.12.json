{"text": "The EC tells the INQ about the change of his master data . The INQ notifies the IP of the change . The IP checks whether the master data can be changed at the desired time . The IP confirmes the changes of the INQ or the IP rejectes the changes of the INQ .", "name": "doc-10.12", "id": "doc-10.12", "category": "", "tokens": [{"text": "The", "indexInDocument": 0, "posTag": "DT", "sentenceIndex": 0}, {"text": "EC", "indexInDocument": 1, "posTag": "NNP", "sentenceIndex": 0}, {"text": "tells", "indexInDocument": 2, "posTag": "VBZ", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 3, "posTag": "DT", "sentenceIndex": 0}, {"text": "INQ", "indexInDocument": 4, "posTag": "NNP", "sentenceIndex": 0}, {"text": "about", "indexInDocument": 5, "posTag": "IN", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 6, "posTag": "DT", "sentenceIndex": 0}, {"text": "change", "indexInDocument": 7, "posTag": "NN", "sentenceIndex": 0}, {"text": "of", "indexInDocument": 8, "posTag": "IN", "sentenceIndex": 0}, {"text": "his", "indexInDocument": 9, "posTag": "PRP$", "sentenceIndex": 0}, {"text": "master", "indexInDocument": 10, "posTag": "NN", "sentenceIndex": 0}, {"text": "data", "indexInDocument": 11, "posTag": "NNS", "sentenceIndex": 0}, {"text": ".", "indexInDocument": 12, "posTag": ".", "sentenceIndex": 0}, {"text": "The", "indexInDocument": 13, "posTag": "DT", "sentenceIndex": 1}, {"text": "INQ", "indexInDocument": 14, "posTag": "NNP", "sentenceIndex": 1}, {"text": "notifies", "indexInDocument": 15, "posTag": "VBZ", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 16, "posTag": "DT", "sentenceIndex": 1}, {"text": "IP", "indexInDocument": 17, "posTag": "NNP", "sentenceIndex": 1}, {"text": "of", "indexInDocument": 18, "posTag": "IN", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 19, "posTag": "DT", "sentenceIndex": 1}, {"text": "change", "indexInDocument": 20, "posTag": "NN", "sentenceIndex": 1}, {"text": ".", "indexInDocument": 21, "posTag": ".", "sentenceIndex": 1}, {"text": "The", "indexInDocument": 22, "posTag": "DT", "sentenceIndex": 2}, {"text": "IP", "indexInDocument": 23, "posTag": "NNP", "sentenceIndex": 2}, {"text": "checks", "indexInDocument": 24, "posTag": "NNS", "sentenceIndex": 2}, {"text": "whether", "indexInDocument": 25, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 26, "posTag": "DT", "sentenceIndex": 2}, {"text": "master", "indexInDocument": 27, "posTag": "NN", "sentenceIndex": 2}, {"text": "data", "indexInDocument": 28, "posTag": "NNS", "sentenceIndex": 2}, {"text": "can", "indexInDocument": 29, "posTag": "MD", "sentenceIndex": 2}, {"text": "be", "indexInDocument": 30, "posTag": "VB", "sentenceIndex": 2}, {"text": "changed", "indexInDocument": 31, "posTag": "VBN", "sentenceIndex": 2}, {"text": "at", "indexInDocument": 32, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 33, "posTag": "DT", "sentenceIndex": 2}, {"text": "desired", "indexInDocument": 34, "posTag": "JJ", "sentenceIndex": 2}, {"text": "time", "indexInDocument": 35, "posTag": "NN", "sentenceIndex": 2}, {"text": ".", "indexInDocument": 36, "posTag": ".", "sentenceIndex": 2}, {"text": "The", "indexInDocument": 37, "posTag": "DT", "sentenceIndex": 3}, {"text": "IP", "indexInDocument": 38, "posTag": "NNP", "sentenceIndex": 3}, {"text": "confirmes", "indexInDocument": 39, "posTag": "VBZ", "sentenceIndex": 3}, {"text": "the", "indexInDocument": 40, "posTag": "DT", "sentenceIndex": 3}, {"text": "changes", "indexInDocument": 41, "posTag": "NNS", "sentenceIndex": 3}, {"text": "of", "indexInDocument": 42, "posTag": "IN", "sentenceIndex": 3}, {"text": "the", "indexInDocument": 43, "posTag": "DT", "sentenceIndex": 3}, {"text": "INQ", "indexInDocument": 44, "posTag": "NNP", "sentenceIndex": 3}, {"text": "or", "indexInDocument": 45, "posTag": "CC", "sentenceIndex": 3}, {"text": "the", "indexInDocument": 46, "posTag": "DT", "sentenceIndex": 3}, {"text": "IP", "indexInDocument": 47, "posTag": "NNP", "sentenceIndex": 3}, {"text": "rejectes", "indexInDocument": 48, "posTag": "VBZ", "sentenceIndex": 3}, {"text": "the", "indexInDocument": 49, "posTag": "DT", "sentenceIndex": 3}, {"text": "changes", "indexInDocument": 50, "posTag": "NNS", "sentenceIndex": 3}, {"text": "of", "indexInDocument": 51, "posTag": "IN", "sentenceIndex": 3}, {"text": "the", "indexInDocument": 52, "posTag": "DT", "sentenceIndex": 3}, {"text": "INQ", "indexInDocument": 53, "posTag": "NNP", "sentenceIndex": 3}, {"text": ".", "indexInDocument": 54, "posTag": ".", "sentenceIndex": 3}], "mentions": [{"type": "activity", "tokenDocumentIndices": [2]}, {"type": "activity", "tokenDocumentIndices": [15]}, {"type": "activity", "tokenDocumentIndices": [24]}, {"type": "activity", "tokenDocumentIndices": [39]}, {"type": "activity", "tokenDocumentIndices": [48]}, {"type": "actor", "tokenDocumentIndices": [0, 1]}, {"type": "actor", "tokenDocumentIndices": [3, 4]}, {"type": "actor", "tokenDocumentIndices": [13, 14]}, {"type": "actor", "tokenDocumentIndices": [22, 23]}, {"type": "actor", "tokenDocumentIndices": [37, 38]}, {"type": "actor", "tokenDocumentIndices": [46, 47]}, {"type": "actor", "tokenDocumentIndices": [43, 44]}, {"type": "actor", "tokenDocumentIndices": [52, 53]}, {"type": "activity data", "tokenDocumentIndices": [6, 7, 8, 9, 10, 11]}, {"type": "activity data", "tokenDocumentIndices": [16, 17]}, {"type": "activity data", "tokenDocumentIndices": [25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35]}, {"type": "activity data", "tokenDocumentIndices": [40, 41, 42, 43, 44]}, {"type": "activity data", "tokenDocumentIndices": [49, 50, 51, 52, 53]}, {"type": "further specification", "tokenDocumentIndices": [25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35]}, {"type": "xor gateway", "tokenDocumentIndices": [45]}, {"type": "condition specification", "tokenDocumentIndices": [45]}], "entities": [{"mentionIndices": []}, {"mentionIndices": [6, 7]}, {"mentionIndices": [8, 9]}, {"mentionIndices": []}, {"mentionIndices": [11, 12]}, {"mentionIndices": []}, {"mentionIndices": []}, {"mentionIndices": []}, {"mentionIndices": [16, 17]}, {"mentionIndices": [0]}, {"mentionIndices": [1]}, {"mentionIndices": [2]}, {"mentionIndices": [3]}, {"mentionIndices": [4]}, {"mentionIndices": [5]}, {"mentionIndices": [10]}, {"mentionIndices": [13]}, {"mentionIndices": [14]}, {"mentionIndices": [15]}, {"mentionIndices": [18]}, {"mentionIndices": [19]}, {"mentionIndices": [20]}], "relations": [{"headMentionIndex": 19, "tailMentionIndex": 20, "type": "same gateway"}, {"headMentionIndex": 0, "tailMentionIndex": 1, "type": "flow"}, {"headMentionIndex": 1, "tailMentionIndex": 2, "type": "flow"}, {"headMentionIndex": 2, "tailMentionIndex": 3, "type": "flow"}, {"headMentionIndex": 3, "tailMentionIndex": 19, "type": "flow"}, {"headMentionIndex": 19, "tailMentionIndex": 20, "type": "flow"}, {"headMentionIndex": 20, "tailMentionIndex": 4, "type": "flow"}, {"headMentionIndex": 0, "tailMentionIndex": 5, "type": "actor performer"}, {"headMentionIndex": 0, "tailMentionIndex": 6, "type": "actor recipient"}, {"headMentionIndex": 0, "tailMentionIndex": 13, "type": "uses"}, {"headMentionIndex": 1, "tailMentionIndex": 7, "type": "actor performer"}, {"headMentionIndex": 1, "tailMentionIndex": 14, "type": "uses"}, {"headMentionIndex": 2, "tailMentionIndex": 8, "type": "actor performer"}, {"headMentionIndex": 2, "tailMentionIndex": 18, "type": "further specification"}, {"headMentionIndex": 2, "tailMentionIndex": 15, "type": "uses"}, {"headMentionIndex": 3, "tailMentionIndex": 9, "type": "actor performer"}, {"headMentionIndex": 3, "tailMentionIndex": 16, "type": "uses"}, {"headMentionIndex": 4, "tailMentionIndex": 10, "type": "actor performer"}, {"headMentionIndex": 4, "tailMentionIndex": 17, "type": "uses"}]}