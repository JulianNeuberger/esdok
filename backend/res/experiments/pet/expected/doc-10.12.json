{"nodes": [{"id": "0", "name": "The EC", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "1", "name": "tells", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "2", "name": "the INQ", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "3", "name": "about the change of his master data", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "4", "name": "notifies", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "5", "name": "the IP", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "6", "name": "the changes of the INQ", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "7", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "8", "name": "whether the master data can be changed at the desired time", "entity": {"name": "further specification", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "9", "name": "confirmes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "10", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, {"id": "11", "name": "rejectes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}], "edges": [{"id": "0", "source": {"id": "1", "name": "tells", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "about the change of his master data", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "1", "source": {"id": "1", "name": "tells", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "0", "name": "The EC", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "2", "source": {"id": "1", "name": "tells", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the INQ", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "3", "source": {"id": "1", "name": "tells", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "notifies", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "4", "source": {"id": "4", "name": "notifies", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "6", "name": "the changes of the INQ", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "5", "source": {"id": "4", "name": "notifies", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the INQ", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "6", "source": {"id": "4", "name": "notifies", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "the IP", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "7", "source": {"id": "4", "name": "notifies", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "7", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "8", "source": {"id": "7", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "the IP", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "9", "source": {"id": "7", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "10", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "10", "source": {"id": "7", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "8", "name": "whether the master data can be changed at the desired time", "entity": {"name": "further specification", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "further specification"}, {"id": "11", "source": {"id": "9", "name": "confirmes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "6", "name": "the changes of the INQ", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "12", "source": {"id": "9", "name": "confirmes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "the IP", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "13", "source": {"id": "10", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "11", "name": "rejectes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "14", "source": {"id": "10", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "9", "name": "confirmes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "15", "source": {"id": "11", "name": "rejectes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "6", "name": "the changes of the INQ", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "16", "source": {"id": "11", "name": "rejectes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "the IP", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.12", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}]}